Utils/TimeUtils.cpp

```diff
@@ -4,7 +4,7 @@
 // Add before any ATL header:
 // https://social.msdn.microsoft.com/Forums/vstudio/en-US/371a069b-f428-47b5-8f7b-e9ad85964bce/guidnull-undeclared-identifier?forum=vclanguage
 #include <cguid.h>
-
+#include <ctime>
```

Comment:
Should be before Windows headers for better linkage (we talked about include order over our phone call)
Best practice:
1. Your private headers
2. STL headers
3. Boost headers
4. Windows headers 

There are special cases like the case I've documented in line 6 (with <cguid.h> bug)